benchmarks/tools/benchmark.py

```diff
@@ -0,0 +1,176 @@
+from argparse import ArgumentParser
+from collections import OrderedDict
+from collections.abc import Iterable
+from itertools import product
+from datetime import datetime
+from os import path, makedirs
+from devito.logger import warning
+
+import json
```

Comment:
is json part of the standard library ? if so, it should appear in the group above?

Reply:
fixed